softwerkskammer/testIntegration/app_test.js

```diff
@@ -2,7 +2,7 @@
 var expect = require('must-dist');
 var httpRequest = require('request');
 var sinon = require('sinon').sandbox.create();
-var conf = require('../testutil/configureForTest');
+var conf = require('../testutil/configureForTestWithDB.js');
```

Comment:
Ich verstehe und honoriere deine intention. Dennoch:

1. Ich fürchte, du verrennst dich hier gerade in einen (extrem großen) Nebenkriegsschauplatz
1. Wir haben diesen Tradeoff (das Antipattern) bewußt in Kauf genommen, um eben die Konstruktorinjektion los zu werden. Das war unglaublich unübersichtlich und groß.